libs/feature-workspaces/src/lib/feature-workspaces.module.ts

```diff
@@ -43,6 +43,10 @@ export const workspaceRoutes: Route[] = [
     component: NewWorkspaceComponent,
     data: { state: CREATE_WORKSPACE }
   },
+  {
```

Comment:
We do it twice so that we can do /connect or /workspace/WORKSPACE-PATH/connect.

Until we make some alternations to the electron version of the app this is necessary

Reply:
Can we instead make it such that we always go to /connect. Something like this:

```
     if (this.settings.showConnectPlugin()) {
        this.routes.splice(this.routes.length - 2, 0, {
          icon: 'question_answer',
          url: '/connect',
          title: 'Connect'
        });
      }
```

The connect route should be nested in a workspace. We don't use the workspace information in any way right now.